libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/app/SalesforceHybridSDKManager.java

```diff
@@ -83,9 +83,10 @@ private static void init(Context context, Class<? extends Activity> mainActivity
     		INSTANCE = new SalesforceHybridSDKManager(context, mainActivity, loginActivity);
     	}
 
+        initInternal(context);
```

Comment:
This call is meant to be after the upgrade step, by design. There's a lot of internal initialization that occurs there, and without running upgrade steps, it would lead to permanent corruption. For example, `SmartStore` DB opening calls. Let's move this back below.